tests/postgres_tests/test_constraints.py

```diff
@@ -0,0 +1,37 @@
+from django.db import connection, transaction
+from django.db.models import Q
+from django.db.models.constraints import CheckConstraint
+from django.db.utils import IntegrityError
+
+from . import PostgreSQLTestCase
+from .models import RangesModel
+
+try:
+    from psycopg2.extras import NumericRange
+except ImportError:
+    pass
+
+
+class SchemaTests(PostgreSQLTestCase):
+    def get_constraints(self, table):
+        """
```

Comment:
Seems to fine on a single line.

Reply:
Will change, FWIW this was extracted as is from `test_indexes`

https://github.com/django/django/blob/c18573187ad160d7c9275173e0082863a836648e/tests/postgres_tests/test_indexes.py#L143-L148